src/app/learn/faq/index.html

```diff
@@ -0,0 +1,19 @@
+<app-nav></app-nav>
+<stache 
+  pageTitle="FAQ"
+  layout="sidebar"
+  showBreadcrumbs="true"
+  showTableOfContents="true"
+  [sidebarRoutes]="[{ name: 'Overview', path: '/learn' }, { name: 'Tutorials', path: '/learn/tutorials' }, { name: 'Technical Reference', path: '/learn/reference' }, { name: 'Resources', path: '/learn/resources' }, { name: 'FAQ', path: '/learn/faq' }]"
+  [breadcrumbsRoutes]="[{ name: 'Home', path: '/' }, { name: 'Learn', path: '/learn' }]">
+
+  <stache-page-summary>
+    <p>
+      The SKY UX framework provides a consistent, cohesive user experience for Blackbaud products and for third-party developers. SKY UX provides the components, services, and modules that developers need to incorporate Blackbaud styles and patterns.
+    </p>
+  </stache-page-summary>
+
+  <stache-forward-back-buttons
+    [routes]="[{ name: 'Resources', path: '/learn/resources' }, undefined]">
```

Comment:
I was under the impression that we were going to be explicit with the inputs, `previousRoute` and `nextRoute`?